assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -117,3 +121,18 @@ che.keycloak.github.endpoint=NULL
 
 #     The number of seconds to tolerate for clock skew when verifying exp or nbf claims.
 che.keycloak.allowed_clock_skew_sec=3
+
+#     Use the OIDC optional `nonce` feature to increase security.
+che.keycloak.use_nonce=true
+
+#     URL to the Keycloak Javascript adapter we want to use.
+#     if set to NULL, then the default used value is  
+#     `${che.keycloak.auth_server_url}/js/keycloak.js`,
+#     or `<che-server>/wsmaster/keycloak/OIDCKeycloak.js`
```

Comment:
There is no `/wsmaster` URL's. Correct is `/api`

Reply:
Fixed in commit https://github.com/eclipse/che/pull/8650/commits/8951eef7268c5682b054f9c332ce994efa25ee71